api/src/main/java/marquez/db/OpenLineageDao.java

```diff
@@ -79,6 +80,12 @@ public interface OpenLineageDao extends BaseDao {
   String DEFAULT_SOURCE_NAME = "default";
   String DEFAULT_NAMESPACE_OWNER = "anonymous";
 
+  enum SpecEventType {
+    RUN_EVENT,
+    DATASET_EVENT,
+    JOB_EVENT;
+  }
+
   @SqlUpdate(
       "INSERT INTO lineage_events ("
           + "event_type, "
```

Comment:
_Minor_: We can consider defining a `_run_state` column and eventually dropping the `event_type`.